src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -61,24 +63,36 @@ const ChartsHolder = (props: {
   const icuUtilizationData =
     projection && projection.getDataset('icuUtilization').data;
 
+  const getChartSummarys = (projection: Projection) => {
+    return {
+      [ChartType.CASE_GROWTH_RATE]: projection.getLatestColumnValue('rtRange'),
+      [ChartType.HOSPITAL_USAGE]: projection.getLatestColumnValue(
+        'icuUtilization',
+      ),
+      [ChartType.POSITIVE_TESTS]: projection.getLatestColumnValue(
+        'testPositiveRate',
+      ),
+    };
+  };
+
   return (
     <>
       {!projection ? (
         <NoCountyDetail countyId={props.countyId} stateId={props.stateId} />
       ) : (
         <ChartContentWrapper>
           <LocationPageHeader projections={props.projections} />
-          <SummaryStats stats={[]} />
+          {projection.isInferred && <SummaryStats stats={getChartSummarys(projection)} />}
           <MainContentInner>
             <ChartHeader></ChartHeader>
-            <h1>Case growth</h1>
+            <h1>{ChartType.CASE_GROWTH_RATE}</h1>
```

Comment:
It's kind of strange to use the value of the enum as the website copy.

Reply:
bhahahahah yeah it isss

Reply:
okay i made vars for this